uint8 status
Header header
float64 time
float64 x
float64 y
float64 z
float64 qw
float64 qx
float64 qy
float64 qz
float64 vbx
float64 vby
float64 vbz
float64 omegax
float64 omegay
float64 omegaz
float64 omega1
float64 omega2
float64 omega3
float64 omega4
float64 vx
float64 vy
float64 vz
